skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -369,10 +365,8 @@ def _nl_means_denoising_3d(image, int s=7, int d=13, double h=0.1,
 #-------------- Accelerated algorithm of Froment 2015 ------------------
 
 
-@cython.cdivision(True)
-@cython.boundscheck(False)
-cdef inline double _integral_to_distance_2d(IMGDTYPE [:, ::] integral,
-                        int row, int col, int offset, double h2s2):
+cdef inline double _integral_to_distance_2d(
+        IMGDTYPE [:, ::] integral, int row, int col, int offset, double h2s2):
```

Comment:
Won't the following be more PEP8 compliant, and also clearer (in one similar case below as well):
```
cdef inline double _integral_to_distance_2d(IMGDTYPE [:, ::] integral,
                                            int row, int col, int offset,
                                            double h2s2):

```